fix an issue where calls to exec would crash the app (#113)
cybex-gmbh/laravel-protector: README.md

This package allows you to download, export and import your application's database backups.

> [!IMPORTANT]

> > To use this package, you must not disable "proc_open" in your PHP configuration.

Reviewer: ```suggestion
> This package will not work if you have disabled "proc_open" in your PHP configuration.
```
